<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagsWidget</class>
 <widget class="QWidget" name="TagsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tags</string>
  </property>
  <property name="windowIcon">
   <iconset theme="git-tag">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="treeViewTags">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PushButton" name="pushButtonAddTag">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="pushButtonRemove">
       <property name="text">
        <string>Remove selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="pushButtonCheckout">
       <property name="text">
        <string>Checkout to selected</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>core/pushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
